sl-example-rspec/buildSession4Tests.txt

```diff
@@ -0,0 +1 @@
+975c43fb-d662-4e86-95ce-3991f7e101a0
```

Comment:
I think that this file shouldn't be submitted

Reply:
the readme says that there should be such file, the scripts point to its path -- it'd be only inconvenient for the example's user to additionally have to create the file. why not have it here right away?

Reply:
The empty file is better you can event put into a prompt 'put the build-session-id' or something like this